src/SFML/System/Utils.hpp

```diff
@@ -29,17 +29,28 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <cctype>
+#include <filesystem>
+#include <sstream>
 #include <string>
 
 
 namespace sf
 {
-    [[nodiscard]] inline std::string toLower(std::string str)
-    {
-        for (char& c : str)
-            c = static_cast<char>(std::tolower(static_cast<unsigned char>(c)));
-        return str;
-    }
+[[nodiscard]] inline std::string toLower(std::string str)
+{
+    for (char& c : str)
+        c = static_cast<char>(std::tolower(static_cast<unsigned char>(c)));
+    return str;
+}
+
+[[nodiscard]] inline std::string printDebugPathInfo(const std::filesystem::path& path)
```

Comment:
Having "print" in the name makes me expect this function to write to an output stream. Maybe it could be renamed to remove the "print" part.
An alternative would be to have a functor with a reference to a path as member data which actually prints to an `std::ostream`.

Reply:
Good point. I renamed it to `formatDebugPathInfo`.